app/instance-initializers/ember-i18n.js

```diff
@@ -1,12 +1,24 @@
 import Ember from "ember";
+import Stream from 'ember-i18n/stream';
 import legacyHelper from "ember-i18n/legacy-helper";
 import ENV from '../config/environment';
 
 export default {
   name: 'ember-i18n',
 
-  initialize: function(instance) {
+  initialize: function({ container }) {
```

Comment:
Curiosity: what will it do? container will be applicationInstance.container? Some kind of destructuring? Taking container as argument is deprecated, but you use here `{}` so that's why I'm asking. :) @jamesarosen